config.env

```diff
@@ -13,15 +13,15 @@ AWS_ROUTE53_HOSTEDZONEID=
 # region where secret is hosted, optional when env=local
 SECRET_HOST_REGION=
 
-NODE_DELETION_TIME_IN_SECONDS=500
+NODE_DELETION_TIME_IN_SECONDS=
 
 # required for env=local
 AWS_ACCESS_ID=
 AWS_SECRET_KEY=
 ## optional
 AWS_TOKEN=
 
-AZURE_CLOUD_PROVIDER=AZUREPUBLICCLOUD
+AZURE_CLOUD_PROVIDER=
```

Comment:
can you keep this value in config, this is used for reference.

Reply:
Let's keep the config file empty as these fields anyway need to populate when running locally.

Reply:
this is lot of work when using outside netbook context, we do have config copy. But when external user is using this he may not have the values without looking  up, so I have kept defaults.